packages/calypso-e2e/src/lib/pages/login-page.ts

```diff
@@ -31,6 +31,8 @@ export class LoginPage {
 				status: 200,
 			} );
 		} );
+		// Reload the page to clear the state.
+		await this.page.reload();
```

Comment:
**question:non-blocking** - my interpretation of https://github.com/microsoft/playwright/issues/18137#issuecomment-1282641320 is that the page reload should come after the login process, but before the navigation to the target URL.